code/modules/events/spacevine.dm

```diff
@@ -537,10 +537,11 @@
 	if(event)
 		event.announce_to_ghosts(vine)
 	START_PROCESSING(SSobj, src)
-	vine_mutations_list = list()
-	init_subtypes(/datum/spacevine_mutation/, vine_mutations_list)
-	for(var/datum/spacevine_mutation/mutation as anything in vine_mutations_list)
-		vine_mutations_list[mutation] = max_mutation_severity - mutation.severity // this is intended to be before the potency check as the ideal maximum potency is used for weighting
+	if(!vine_mutations_list)
+		vine_mutations_list = list()
+		init_subtypes(/datum/spacevine_mutation/, vine_mutations_list)
+		for(var/datum/spacevine_mutation/mutation as anything in vine_mutations_list)
+			vine_mutations_list[mutation] = max_mutation_severity - mutation.severity // this is intended to be before the potency check as the ideal maximum potency is used for weighting
```

Comment:
this is assuming max_mutation_severity does not change, which it might, so I don't support this solution either.

Reply:
Have 'max_mutation_severity' as a DEFINE in this case, so it's more effort for people to change the max mutation severity

Reply:
> this is assuming max_mutation_severity does not change, which it might, so I don't support this solution either.

max_mutation_severity here is meant to not change, as the values are meant to be balanced with the ideal maximum severity, but yeah changing it to a define with a better name would make it more obvious

Reply:
Since the probability of a given mutation appearing is meant to be independent of the kudzu's stats, could you clarify the `using variables that can be modified` part, since these mutation probability values shouldn't change between different kudzu controllers

Reply:
@Watermelon914